### PR TITLE
Remove unused dependencies and configs

### PR DESCRIPTION
**Pull request by heofs**
### What is the context of this PR?
Remove some unused packages from dependencies. If there is any of them you think we should be keeping, then I can add them back in.

I removed the sass loading from webpack as we dont use .scss or .sass files, but I could add it back in if you think it would be good to have in case we want to use it later on?

### How to review 
Check that we do not rely on any of the packages.

### CODE REVIEW

File: eq-author/package.json [hunk 1]
```diff
@@ -93,17 +93,14 @@
     "draft-js-plugins-editor": "latest",
     "draft-js-raw-content-state": "latest",
     "draftjs-filters": "^1.0.0",
-    "enzyme-to-json": "latest",
```
Comment (tgandrews) [inline eq-author/package.json]: This is used by jest and defined in `package.json` line 150

File: eq-author/package.json [hunk 2]
```diff
@@ -38,6 +38,7 @@
     "dotenv-expand": "latest",
     "enzyme": "latest",
     "enzyme-adapter-react-16": "latest",
+    "enzyme-to-json": "^3.3.5",
```
Comment (tgandrews) [inline eq-author/package.json]: Can this target `latest`

Comment (heofs) [reply eq-author/package.json]: Changed it to latest

File: eq-author/package.json [hunk 3]
```diff
@@ -93,19 +93,15 @@
     "draft-js-plugins-editor": "latest",
     "draft-js-raw-content-state": "latest",
     "draftjs-filters": "^1.0.0",
-    "enzyme-to-json": "latest",
     "firebase": "latest",
     "firebaseui": "latest",
     "graphql": "latest",
     "graphql-anywhere": "latest",
-    "graphql-iso-date": "latest",
```
Comment (samiwel) [inline eq-author/package.json]: Don't we still use this? https://github.com/ONSdigital/eq-author-app/blob/master/eq-author-api/schema/resolvers/base.js#L1

Comment (heofs) [reply eq-author/package.json]: This is for eq-author, eq-author-api `package.json` still has `graphql-iso-date` as dependency.

Comment (heofs) [reply eq-author/package.json]: This is for eq-author, eq-author-api `package.json` still has `graphql-iso-date` as dependency.

**Pull request merged**